tests/integration_test/services_test.py

```
@@ -12988,35 +13104,68 @@ def _mock_get_bot_settings(*args, **kwargs):
         f"/api/bot/{pytest.bot}/action/prompt",
         headers={"Authorization": pytest.token_type + " " + pytest.access_token},
     )
-    actual = response.json()
-    actual["data"][3].pop("_id")
-    assert not DeepDiff(actual["data"][3], {
-        'name': 'test_add_prompt_action_with_bot_content_prompt_with_content', 'num_bot_responses': 5,
+    actual=response.json()
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)